docs/quick-start.md

```diff
@@ -1,28 +1,176 @@
+numalogic-simple-pipeline-daemon-66bbd94c4-hf4k2   1/1     Running   0          46s
+numalogic-simple-pipeline-inference-0-n3asg        2/2     Running   0          46s
+numalogic-simple-pipeline-postprocess-0-bw67q      2/2     Running   0          46s
+numalogic-simple-pipeline-out-0-hjb7m              1/1     Running   0          46s
+numalogic-simple-pipeline-in-0-tmd0v               1/1     Running   0          46s
+```
+### Sending data to the pipeline
+
+Once the pipeline has been created, the data can be sent to the pipeline by port-forwarding the input vertex.
+
+1. Port-forward to the http-source vertex
+   ```
+   kubectl port-forward simple-numalogic-pipeline-in-0-xxxxx 8443
+   ```
+   
+2. Send the data to the pod via curl
+   ```
+   curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[0.9,0.1,0.2,0.9,0.9,0.9,0.9,0.8,1,0.9,0.9,0.7]}'
+   ```
+   Note: only send an array of length 12 in data, as the sequence length used for training is 12.   
+
+   
+### Training
 
-## Using Jupyter Notebook:
+Initially there is no ML model present, to trigger training do a curl command and send any data to the pipeline. 
+
+The following logs will be seen in the training pod.
 
-Start Jupyter Notebook, navigate to the examples folder and open the quick start Jupyter notebook. You can play with different available options for each step.
 ```shell
-jupyter Notebook .
-```
\ No newline at end of file
+> curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[0.9,0.1,0.2,0.9,0.9,0.9,0.9,0.8,1,0.9,0.9,0.7]}'
+
+> kubectl logs numalogic-simple-pipeline-train-0-xxxxx -c udf
+2022-10-19 22:38:45,431 - INFO - Training autoencoder model..
+2022-10-19 22:38:45,783 - INFO - epoch : 5, loss_mean : 0.0744678
+2022-10-19 22:38:46,431 - INFO - epoch : 10, loss_mean : 0.0491540
+...
+2022-10-19 22:38:49,645 - INFO - epoch : 95, loss_mean : 0.0047888
+2022-10-19 22:38:49,878 - INFO - epoch : 100, loss_mean : 0.0043651
+2022-10-19 22:38:49,880 - INFO - 8b597791-b8a3-41b0-8375-47e168887c54 - Training complete
+Successfully registered model 'ae::model'.
+2022/10/19 22:38:52 INFO mlflow.tracking._model_registry.client: Waiting up to 300 seconds for model version to finish creation.                     Model name: ae::model, version 1
+Created version '1' of model 'ae::model'.
+2022-10-19 22:38:52,920 - INFO - 8b597791-b8a3-41b0-8375-47e168887c54 - Model Saving complete
+```
+
+### Inference
+
+Now, the pipeline is ready for inference with the model trained above, data can be sent to the pipeline for ML inference. 
+
+In the output, `ts_data` is the final array that the input array has been transformed to, after all the steps in the pipeline. `anomaly_score` is the final anomaly score generated for the input data.
+
+Sending non-anomalous data: 
+```
+curl -kq -X POST https://localhost:8443/vertices/in -d '{"data":[358.060687,326.253469,329.023996,346.168602,339.511273,359.080987,341.036110,333.584121,376.034150,351.065394,355.379422,333.347769]}'
+
```

Comment:
call out that the logs are from the output vertex. this looks like `curl`'s response